### ISSUE TITLE
minor wording update needed

### ISSUE THREAD

**Issue by masukomi** [federico-terzi/espanso#2110]
### Description

In the wizard that guides new macOS users through the setup it includes a screenshot that says "Open System Preferences" when talking about enabling accessibility. This is now called "System Settings" in macOS Sonoma 

### Steps to reproduce

1. use the setup wizard on macOS sonoma. 
2. navigate to the page about enabling accessibility

### Expected behavior

screenshot would match dialog 

### Screenshots

![CleanShot 2024-10-26 at 10 25 23@2x](https://github.com/user-attachments/assets/50d082ea-bb17-43f6-ad7e-62ee4e9cdc15)

### Your environment

- Public source code: unused
- Environment name and version (e.g., Rust 1.59.0): not applicable? 
- Operating system and version (e.g., Ubuntu 20.04.2 LTS): macos Sonoma 14.6.1
- Espanso version (e.g., 2.1.4-beta): 2.2.1

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by smeech** [comment]
I think this refers to https://github.com/espanso/espanso/blob/16e342fcb43a2c99e25c9b7ae85642b0010054f9/espanso-modulo/src/sys/wizard/wizard.fbp#L3252? It may need different wording, although I presume the current wording is correct for some versions of macOS?

And the image is: https://github.com/espanso/espanso/blob/dev/images/accessibility-prompt.png

**Comment by Archigos** [comment]
@smeech  You're correct, that's where the code for this is... and the original post is also correct that Apple has changed the name of the System Settings/System Preferences...  I haven't had a chance to look at the surrounding code yet, but it'd probably be one of two ways to 'fix' this...

*  Right before the dialog is shown, detect macOS Version number and pick the proper name based on version
*  Reword it to say something like "Open System Preferences/Settings" or something like that to make it clear it might have multiple names.  As far as an updated Screenshot, I can take one from Sequoia (I believe it should look identical to Sonoma) and we could maybe show both if it's in the docs on the site... Like "Ventura and Earlier" (the screenshot from above) "Sonoma and Later" (the new screenshot I'll send)

**Comment by smeech** [comment]
I don't believe the [docs](https://espanso.org/docs/install/mac) mention it at all - perhaps it's left to the wizard to guide people through.

A rewording in the code will probably work, but that's not really my field.

**Comment by Archigos** [comment]
The easiest thing, we just need to update 3 images in [images](https://github.com/espanso/espanso/tree/dev/images)... but ideally (maybe at a later time) we can do the Version detected as mentioned above, and show the proper images based on System... Apple doesn't change the names/locations often, but when they do it's usually a 'massive' change... enough so that a lot of people would be confused since it's just 'oh, it's 2 lower in the list then before' or something simple.
